Documentation/common-problems.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Common Problems
```

Comment:
Thanks for putting this together! My comments are pretty much minor formatting suggestions. The content looks great.